packages/KPagination/KPagination.vue

```diff
@@ -374,16 +375,20 @@ export default {
     align-self: center;
     width: 32px;
     height: 32px;
-
-    line-height: 36px;
+    line-height: 20px;
+    font-size: 12px;
     font-weight: initial;
     color: var(--grey-500);
-    border: 1px solid var(--grey-200);
+    border: 1px solid var(--grey-300);
     background-color: white;
     border-radius: 4px;
     margin: 0 6px;
     cursor: pointer;
 
+    a, div {
+      padding: 6px;
+    }
```

Comment:
I think this rule, and wrapping the `...` in a `div` needs to be ported to `beta` branch @kaiarrowood